.github/workflows/project.yml

```diff
@@ -86,8 +86,8 @@ jobs:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
       - uses: ./.github/actions/node/latest
       - uses: ./.github/actions/install
-      - run: yarn type:test
-      - run: yarn type:doc
+      - run: npm run type:test
+      - run: npm run type:doc
```

Comment:
Is this necessary? It's inconsistent w/ the rest of our workflows. Maybe do this in a separate PR?

Reply:
It is not necessary. I just stumbled over it and we spoke about changing that as well. I understand that it's nicer to separate it out, while I thought we want to change this anyway, so it should not hurt either?